chain/near/src/trigger.rs

```diff
@@ -175,7 +176,8 @@ mod tests {
             receipt: receipt().unwrap(),
         }));
 
-        let result = blockchain::MappingTrigger::to_asc_ptr(trigger, &mut heap, &GasCounter::new());
+        let result =
+            blockchain::MappingTrigger::to_asc_ptr(trigger, &mut heap, &GasCounter::default());
```

Comment:
I disagree with clippy here, nothing wrong with a `new` method that does the same thing as `default`.

Reply:
Yeah... the reasoning behind Clippy's suggestion is that `Default` integrates better with `std::mem::take`, `unwrap_or_default` and such, but I guess it's a preference thing, really.

Reply:
I'll define `GasCounter::new` as `GasCounter::default`, as to have both.